doc/api/net.md

```diff
@@ -281,6 +281,26 @@ added: v0.1.90
 
 Emitted when the server has been bound after calling [`server.listen()`][].
 
+### Event: `'drop'`
+
```

Comment:
Could we add the data property description? something like that:

```
* `data` {Object} The argument passed to event listener.
  * `localAddress` {string}  Local address.
  * `localPort` {number} Local port.
  * `remoteAddress` {string} Remote address.
  * `remotePort` {number} Remote port.
  * `remoteFamily` {string} Remote IP family. `'IPv4'` or `'IPv6'`.
```

Reply:
> ```gfm
> * `data` {Object} The argument passed to event listener.
>   * `localAddress` {string}  Local address.
>   * `localPort` {number} Local port.
>   * `remoteAddress` {string} Remote address.
>   * `remotePort` {number} Remote port.
>   * `remoteFamily` {string} Remote IP family. `'IPv4'` or `'IPv6'`.
> ```

Thanks ! Done !